x/withdraw/types/params.go

```diff
@@ -0,0 +1,76 @@
+package types
+
+import (
+	"fmt"
+	"time"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// Parameter store key
+var (
+	ParamStoreKeyEnableWithdraw        = []byte("EnableWithdraw")
+	ParamStoreKeyPacketTimeoutDuration = []byte("PacketTimeoutDuration")
+)
+
+// ParamKeyTable returns the parameter key table.
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
```

Comment:
why return empty `KeyTable`? Is this unused boilerplate?

Reply:
it's not empty, it registers the parameters to the key table. Used in the keeper